e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -23,25 +20,53 @@ const openDeepLink = async (payUrl) => {
 
 export default HandleDeepLinkSend = () => {
   describe('When Launching Deeplink - App Closed', () => {
+    let commentText
     it('Then should handle deeplink with all attributes', async () => {
-      const PAY_URL = quote(deepLinks.withAll)
+      commentText = `${new Date().getTime()}-${parseInt(Math.random() * 100000)}`
```

Comment:
@MuckT can you use a fixed comment instead? we often just rerun e2e tests when they fail instead of looking closely at what went wrong. so if this failed only some of the time, we could entirely miss it and just think "what a flaky test". in general, I am not a fan of tests that rely on randomness for this reason (also because it can be hard to repro the failure)